src/main/java/uk/gov/hmcts/reform/pip/data/management/service/AccountManagementService.java

```diff
@@ -42,4 +48,33 @@ public Boolean getIsAuthorised(UUID userId, ListType listType, Sensitivity sensi
         }
     }
 
+    public List<String> getAllAccounts(String pageNumber, String pageSize, String provenances)
+        throws JsonProcessingException  {
+        try {
+            String result = webClient.get().uri(String.format(
+                    "%s/account/all?pageNumber=%s&pageSize=%s&provenances=%s", url, pageNumber, pageSize, provenances))
```

Comment:
feel like we should return all system admin users directly from account management rather than retrieve all then filtering the results. You can pass "roles" as a query param in this endpoint. Also why do we need pageNumber and pageSize? Don't we want all SYSTEM_ADMIN rather than those in early pages?

Reply:
I have changed the api call to return only systemAdmins.